nengo/params.py

```diff
@@ -416,7 +417,7 @@ def validate(self, instance, function_info):
         super(FunctionParam, self).validate(instance, function)
 
 
-class FrozenObject(object):
+class FrozenObject(with_metaclass(DocstringInheritor)):
```

Comment:
> The alternative is to do multiple inheritance in the solver; i.e., class Solver(FrozenObject, with_metaclass(DocstringInheritor)):.

Shouldn't it be `Solver(with_metaclass(DocstringInheritor, FrozenObject)):`? At least I believe that is what I used in some of my PRs.